spec/index.bs

```diff
@@ -8263,8 +8263,8 @@ enum GPUBlendOperation {
 dictionary GPUDepthStencilState {
     required GPUTextureFormat format;
 
-    boolean depthWriteEnabled = false;
-    GPUCompareFunction depthCompare = "always";
+    required boolean depthWriteEnabled = false;
+    required GPUCompareFunction depthCompare = "always";
```

Comment:
Remove the defaults from both of these now that they're required.

Reply:
Good catch. Updated, thanks.